Lib/test/test_pprint.py

```diff
@@ -346,6 +346,36 @@ def test_mapping_proxy(self):
                           ('lazy', 7),
                           ('dog', 8)]))""")
 
+    def test_empty_simple_namespace(self):
+        self.assertEqual(pprint.pformat(types.SimpleNamespace()), "namespace()")
+
+    def test_small_simple_namespace(self):
+        n = types.SimpleNamespace(a=1, b=2)
+        self.assertEqual(pprint.pformat(n), "namespace(a=1, b=2)")
+
+    def test_simple_namespace(self):
+        n = types.SimpleNamespace(
+            the=0,
+            quick=1,
+            brown=2,
+            fox=3,
+            jumped=4,
+            over=5,
+            a=6,
+            lazy=7,
+            dog=8,
+        )
+        self.assertEqual(pprint.pformat(n, width=60), """\
```

Comment:
As before, I recommend one action per line.

```suggestion
        formatted = pprint.pformat(ns, width=60)
        self.assertEqual(formatted, """\
```

...and separating the key sections of the test.

```suggestion

        formatted = pprint.pformat(ns, width=60)

        self.assertEqual(formatted, """\
```